### PR TITLE
[groovyscripting] Described a way for code reuse

### PR DESCRIPTION
**Pull request by OlegAndreych**

### CODE REVIEW

File: bundles/org.openhab.automation.groovyscripting/README.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
+## Code reuse
+
+One can place *.groovy files with groovy classes under `automation/groovy` configuration directory.
```
Comment (wborn) [inline bundles/org.openhab.automation.groovyscripting/README.md:26]:
```suggestion
One can place *.groovy files with Groovy classes under `automation/groovy` configuration directory.  
```

File: bundles/org.openhab.automation.groovyscripting/README.md [hunk 2]
```diff
@@ -21,6 +21,13 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
+## Code reuse
+
+One can place *.groovy files with groovy classes under `automation/groovy` configuration directory.  
+Those classes can be imported in JSR-223 scripts or UI rules action with usual Groovy `import` statement.
```
Comment (wborn) [inline bundles/org.openhab.automation.groovyscripting/README.md:27]:
```suggestion
Those classes can be imported in JSR-223 scripts or the UI rules action with the usual Groovy `import` statement.
```

File: bundles/org.openhab.automation.groovyscripting/README.md [hunk 3]
```diff
@@ -21,6 +21,13 @@ log:set DEBUG org.openhab.core.automation
...
+
+To apply shared code changes, one has to restart `openHAB Core :: Bundles :: Automation` or an OpenHAB instance altogether.
```
Comment (wborn) [inline bundles/org.openhab.automation.groovyscripting/README.md:29]:
```suggestion
To apply shared code changes, one has to restart the `openHAB Core :: Bundles :: Automation` bundle on the Console or an openHAB instance altogether.
```

**Review by wborn — COMMENTED** [review]
Thanks for the extra docs!
I think it can be improved a little bit:

**Comment by OlegAndreych** [comment]
> Thanks for the extra docs! I think it can be improved a little bit:

Agreed

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**